src/state/experiments.test.js

```diff
@@ -110,4 +220,144 @@ describe('experiments/actions', () => {
       });
     });
   });
+
+  describe('create', () => {
+    it('success', () => {
+      const store = mockStore({ experiments: initialState });
+
+      fetch.mockResponseOnce(JSON.stringify({ data: mockRes }));
+
+      store.dispatch(create('new_name', 'new_desc')).then(() => {
+        expect(store.getActions()).toEqual([
+          {
+            type: CREATE,
+          },
+          {
+            type: CREATE_SUCCESS,
+          },
+          {
+            type: LOAD,
+          },
+        ]);
+      });
+    });
+
+    it('error', () => {
+      const store = mockStore({ experiments: initialState });
+
+      const errText = 'some error';
+      fetch.mockReject(errText);
+
+      store.dispatch(create('new_name', 'new_desc')).catch(() => {
```

Comment:
problem with this test: if the action doesn't through any error, test still pass.

Reply:
Fixed: https://github.com/src-d/code-annotation/pull/195/commits/435f0865210bb6fdd571cfc5cb4d75670bb5f25d